jump-core/src/main/java/com/bitdecay/jump/collision/ContactListener.java

```diff
@@ -0,0 +1,11 @@
+package com.bitdecay.jump.collision;
+
+import com.bitdecay.jump.BitBody;
+
+/**
+ * Created by Monday on 9/29/2015.
+ */
+public interface ContactListener {
+    void contactStarted(BitBody other);
+    void contactEnded(BitBody other);
```

Comment:
It is a minor thing and would require maybe a disproportionate amount of coding to handle, but if you break ContactListener up into ContactStartedListener and ContactEndedListener each with their own void method, you can use the Java8 -> syntax for defining the listeners.  Just a thought.